cli/azd/pkg/auth/azd_credential.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package auth
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/public"
+)
+
+var _ azcore.TokenCredential = &azdCredential{}
```

Comment:
Correct.  I should be able to remove it now, however, since there's code in this package that ends up trying to return `azcore.TokenCredential` as a `&azdCredential`.  This is helpful for cases where there's no other code in your package that treats your object as the interface you are trying to implement, and so you want something that will cause the compiler to yell at you if you get the signature of a method wrong.